### DISCUSSION TITLE
Match reverse alias to alias

### DISCUSSION THREAD

**Discussion by the1poet** [simple-login/app#898] [Q&A]
Hi,
I recently signed up for simplelogin and have been during the arduous task of going through my logins and converting them to aliases.
No sooner had I done this, I've already received spam making me think one of the sites I've registered to has been breached.
Issue is, they bcc'd me, so I only know the alias they sent the email to.
How do I search for the reverse alias and match it to the address they sent it to?

**Comment by nguyenkims** [comment]
@the1poet this is indeed something we realized recently as well. You can display the email header and look for the `X-SimpleLogin-Envelope-To` header which should be the alias. In the next version, we'll include the alias in the To: field if it isn't included there.

**Reply by nguyenkims** [reply]
@the1poet this should be addressed in this pull request https://github.com/simple-login/app/pull/899

**Reply by the1poet** [reply]
> @the1poet this is indeed something we realized recently as well. You can display the email header and look for the `X-SimpleLogin-Envelope-To` header which should be the alias. In the next version, we'll include the alias in the To: field if it isn't included there.

Thank you. I managed to find the email address and disable the alias.
